### DISCUSSION TITLE
BuildTool / Sampler

### DISCUSSION THREAD

**Discussion by raandree** [aldrichtr/infraspective#41] [Ideas]
Hi Tim, may I propose to use [Sampler](https://github.com/gaelcolas/Sampler) for the module scaffolding and CI/CD integration. The DSC Community uses it in 50+ modules and I have successfully used it in many projects as well. It adds almost all best practices and patterns learned in the PowerShell community and uses Invoke-Build like your [BuildTool](https://github.com/aldrichtr/BuildTool). The build process can be easily configured and so far Sampler turned out to be very flexible and extensible (like with [Sampler.DscPipeline](https://github.com/SynEdgy/Sampler.DscPipeline)).

**Comment by aldrichtr** [comment]
Hi Raimund! Thank you for letting me know about Sampler.  You're right, there are many features there that seem to be similar to my BuildTool.  I'll definitely look into migrating to it.

**Reply by aldrichtr** [reply]
#47
